meinberlin/apps/maptopicprio/views.py

```diff
@@ -143,6 +143,7 @@ def get_permission_object(self):
 class MapTopicDeleteView(
     mixins.DashboardBaseMixin,
     mixins.DashboardComponentMixin,
+    mixins.DashboardComponentFormSignalMixin,
     mixins.DashboardComponentDeleteSignalMixin,
```

Comment:
why keeping both signals?

Reply:
my thought was that if we remove it we won't get any warning if there's still js code calling `delete`

Reply:
but I'm also ok with removing it, any preference?

Reply:
OK, but where do we look for the warnings? Only if we keep them in a log file.

Reply:
they should be in the server logs (/var/log/django/django.log)